tst/worksheet.tst

```diff
@@ -0,0 +1,62 @@
+#############################################################################
+##
+##  AutoDoc package
+##
+##  Test the behavior of AutoDocWorksheet
+##
+##  Copyright 2018
+##    Contributed by Glen Whitney, studioinfinity.org
+##
+## Licensed under the GPL 2 or later.
+##
+#############################################################################
+
+gap> START_TEST( "AutoDoc package: worksheet.tst" );
+gap> tmpdir := DirectoryTemporary();;
+gap> SetInfoLevel( InfoGAPDoc, 0 );
+gap> AutoDocWorksheet( "tst/worksheet.tst", rec( dir := tmpdir ) );
```

Comment:
This seems to require that the tst file is run while the package dir is the current directory. But that's not necessarily true. The correct portable solution would normally be to use something like `Filename( DirectoriesPackageLibrary( "AutoDoc", "tst" ), "worksheet.tst")` instead -- but of course you want to test the relative filename here.

Perhaps one could add a `LoadPackage("io")` here and then use its `ChangeDirectoryCurrent` command to change to the "right" directory (and change back afterwards, to not risk breaking other tests).